autopush/router/simple.py

```diff
@@ -137,9 +145,27 @@ def route_notification(self, notification, uaid_data):
             returnValue(RouterResponse(response_body="Delivered"))
         else:
             self.metrics.increment("router.broadcast.miss")
+            if self.udp is not None:
+                yield deferToThread(self._send_udp_wake,
+                                    self.udp)
             returnValue(RouterResponse(202, "Notification Stored"))
 
-    #############################################################
+    def _send_udp_wake(self, udp_info):
+        host = udp_info.get("wakeup_host").get("ip")
+        port = udp_info.get("wakeup_host").get("port")
+        data = json.dumps(udp_info.get("mobilenetwork", {}))
+        if port is not None:
+            host = "%s:%d" % (host, port)
+        response = requests.post(
```

Comment:
Ah, figured the host was for load balancing on the provider side. Having a single point is fine, but might make doing multiple provider UDP wake support a bit more of a challenge